### ISSUE TITLE
Implement sorting options

### ISSUE THREAD

**Issue by AsavarTzeth** [owncloud/music#429]
Personally I would prefer if the default sorting was based on album artist (like most music players) instead of artist. But others might disagree, for example one recently requested sorting by composer (#427).

It would be preferable if these were options that could be set by the user.
- Album Artist
- Composer

**Comment by smartroad** [comment]
I agree, many of my albums are now split up making for an untidy interface as there are some tracks which have more than one artist. Having it sort by album artist rather than track artist would be awesome :grin:

**Comment by paulijar** [comment]
The sorting and main-level categories have been based on the album artist since  #503 (released in v0.3.12). The user-selectable  sort order would require quite major changes and will probably not happen anytime soon.

**Comment by paulijar** [comment]
@LeoSum8 
At least the issue https://github.com/owncloud/music/issues/903 is somewhat related to your request, and I *think* that similar things have been requested in some other issue, too.

More likely than Albums view getting a customizable order is that there would be a new view listing the tracks based on the release date or time added. But that's not yet on the roadmap, either.

Note that the Smartlist feature allows you to create random playlists consisting of recently added tracks. And the Advanced Search functionality allows you to find tracks, albums, and more based on many criteria, including the release date and the add date.

**Comment by LeoSum8** [comment]
Thank you for taking the time to explain!

**Comment by paulijar** [comment]
Kamino closed and cloned this issue to [nc-music/music](https://github.com/nc-music/music/issues/18)
